website/docs/docs/guides/guides-overview.md

```diff
@@ -1,6 +1,6 @@
 ---
-sidebar_position: 7
-title: Guides
+sidebar_position: 1
+title: Guides Overview
 ---
 
 Here you can find some guides or recipes on using the Sandpack in different situations. These examples are basic implementations, and you need to consider improving for production, but you can find more detailed guides in the [advanced usage](/advanced-usage/provider) section.
```

Comment:
Can you add here a link to the page? What do you think?

Reply:
Good idea!